cli.js

```diff
@@ -69,15 +101,17 @@ function init(files) {
 		];
 	}
 
-	return globby(files).then(function (files) {
-		files.forEach(function (file) {
-			run(path.resolve(file));
-		});
+	return globby(files)
+		.then(function (files) {
+			// convert relative paths to absolute
+			files = files.map(function (file) {
+				return path.join(process.cwd(), file);
+			});
 
```

Comment:
I think we should first resolve all paths before we do anything. That would make the code cleaner and less coupled. Then with that array of files, run fork (`fork(file).on('message', test)`) on them here.